Formula/subversion.rb

```diff
@@ -16,7 +16,6 @@ class Subversion < Formula
   deprecated_option "perl" => "with-perl"
   deprecated_option "ruby" => "with-ruby"
 
-  option :universal
```

Comment:
```
    resource("serf").stage do
      # SConstruct merges in gssapi linkflags using scons's MergeFlags,
      # but that discards duplicate values - including the duplicate
      # values we want, like multiple -arch values for a universal build.
      # Passing 0 as the `unique` kwarg turns this behaviour off.
      inreplace "SConstruct", "unique=1", "unique=0"
```
This is probably no longer needed unless there are other duplicates. If it is still needed, the comment should be updated.